inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -152,7 +153,6 @@ protected function get_raw_response_data( HttpQueryInterface $query, array $inpu
 
 		return [
 			'metadata' => [
-				'age' => intval( $response->getHeaderLine( 'Age' ) ),
```

Comment:
I noticed that this is still being referenced in `QueryRunner#get_response_metadata`, just need to remove that.

Reply:
Opted to restore it for now in https://github.com/Automattic/remote-data-blocks/pull/475/commits/64549e95e4e01c8b2faa936ef19c35167a1fc354